oolab/src/main/java/agh/ics/oop/model/Animal.java

```diff
@@ -0,0 +1,56 @@
+package agh.ics.oop.model;
+
+import agh.ics.oop.model.Vector2d;
+import agh.ics.oop.model.MapDirection;
+import agh.ics.oop.model.MoveDirection;
+
+public class Animal {
+    private Vector2d position;
+    private MapDirection orientation;
+
+    public Vector2d getPosition() {
+        return position;
+    }
+    public MapDirection getOrientation() {
+        return orientation;
+    }
+
+    public Animal() {
+        this.orientation = MapDirection.NORTH;
+        this.position = new Vector2d(2, 2);
```

Comment:
ten konstruktor jest szczególnym przypadkiem drugiego konstruktora - można to zwinąć korzystając z instrukcji `this(...)` i przekazując do niej odpowiedni parametr.